test/functional/uiautomator-e2e-specs.js

```diff
@@ -0,0 +1,34 @@
+// transpile :mocha
+
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import 'mochawait';
+import UiAutomator from '../../lib/uiautomator.js';
```

Comment:
Can check it like this https://github.com/appium/appium-doctor/blob/master/test/index-specs.js